### ISSUE TITLE
Do not require income/expense/stock accounts in Inventory Config if enable_stock_accounting is false

### ISSUE THREAD

**Issue by jniles** [third-culture-software/bhima#4525]
We have an option to disable "automatic stock accounting" in our [Enterprise Settings](https://github.com/IMA-WorldHealth/bhima/blob/master/client/src/modules/enterprises/enterprises.html#L276).   This makes it so you don't need to worry about accounts if all you actually need is stock management.  It's very useful for the several projects that do not need the value of stock - they just want to keep track of expiration dates and lot numbers.

However you do have to worry about the inventory group accounts for some reason.  Based on our experience, you don't need these accounts.  All installations that want to use BHIMA for stock management only do not ever invoice items.  Therefore, we should remove this requirement.  

Basically:
  1. If enable_auto_stock_accounting is 1, require all accounts for the inventory_group
  2. If enable_auto_stock_accounting is 0, do not require the accounts.

We should also change [the default value](https://github.com/IMA-WorldHealth/bhima/blob/master/server/models/schema.sql#L575) to 1.
